auth-web/src/components/auth/manage-business/AffiliatedEntityList.vue

```diff
@@ -111,6 +170,15 @@ export default class AffiliatedEntityList extends Vue {
     return corpType === CorpType.NAME_REQUEST
   }
 
+  private isApprovedForIA (business: Business): boolean {
+    // Split string tokens into an array to avoid false string matching
+    const supportedEntityFlags = LaunchDarklyService.getFlag(LDFlags.IaSupportedEntities)?.split(' ')
```

Comment:
Is the default value falsey if LD can't get this flag (LD id not configured or network issue)?

Reply:
For sure, with optional chaining it would return `undefined` ie a falsey value in that event.

Reply:
OK, and you obviously handle that 👍